app/controllers/users_controller.rb

```diff
@@ -23,6 +23,21 @@ def show
     end
   end
 
+  def edit
```

Comment:
Nice job on this. We can pull this out to a nice private helper if we want as well.